README

```diff
@@ -1,20 +1,21 @@
-usage : tty-clock [-iuvsScbtrahDBxn] [-C [0-7]] [-f format] [-d delay] [-a nsdelay] [-T tty] 
-    -s            Show seconds                                   
-    -S            Screensaver mode                               
-    -x            Show box                                       
-    -c            Set the clock at the center of the terminal    
-    -C [0-7]      Set the clock color                            
-    -b            Use bold colors                                
-    -t            Set the hour in 12h format                     
-    -u            Use UTC time                                   
-    -T tty        Display the clock on the specified terminal    
-    -r            Do rebound the clock                           
-    -f format     Set the date format                            
-    -n            Don't quit on keypress                         
-    -v            Show tty-clock version                         
-    -i            Show some info about tty-clock                 
-    -h            Show this page                                 
-    -D            Hide date                                      
-    -B            Enable blinking colon                          
-    -d delay      Set the delay between two redraws of the clock. Default 1s. 
+usage : tty-clock [-iuvsScbtrahDBxnz] [-C [0-7]] [-f format] [-d delay] [-a nsdelay] [-T tty]
+    -s            Show seconds
+    -S            Screensaver mode
+    -x            Show box
+    -c            Set the clock at the center of the terminal
+    -z            Set the clock at the bottom right of the terminal
+    -C [0-7]      Set the clock color
+    -b            Use bold colors
+    -t            Set the hour in 12h format
+    -u            Use UTC time
+    -T tty        Display the clock on the specified terminal
+    -r            Do rebound the clock
+    -f format     Set the date format
+    -n            Don't quit on keypress
+    -v            Show tty-clock version
+    -i            Show some info about tty-clock
+    -h            Show this page
+    -D            Hide date
+    -B            Enable blinking colon
+    -d delay      Set the delay between two redraws of the clock. Default 1s.
```

Comment:
this diff is weird - it shouldn't touch all those lines, only add one.

Reply:
Not sure. My original pacthes wouldn't work so I had to add some lines manually. I'll have a look at it later and fix all I can find.